### DISCUSSION TITLE
White space of FormGroupTitle in FormGroup when titleText is not given

### DISCUSSION THREAD

**Discussion by sadie100** [UI5/webcomponents-react#5182] [Q&A]
Hello, I'm in the process of migrating ui5/webcomponents, from a version three years ago to the latest version.

It seems that in the old webcomponents version, there was no additional margin for the title if I don't give title in FormGroup.
However, after migration, whenever I use FormGroup, a \<h6\> tag with the FormGroupTitle class is included and it creates extra margin.

Is this an intended feature or a bug?
If it's intended feature, is there any way to use FormGroup without including this FormGroupTitle component, so that there is no margin above?

I've attached an example created on CodeSandbox.
https://codesandbox.io/s/clever-thunder-w2yf36?file=/src/App.js

**Comment by Lukas742** [comment]
Hi @sadie100 

I've created an [issue](https://github.com/SAP/ui5-webcomponents-react/issues/5197) from this discussion, so we can better link it to a Pull Request.

This [PR](https://github.com/SAP/ui5-webcomponents-react/pull/5196) will make the `textContent` optional :)

**Reply by sadie100** [reply]
Thanks!
